core/network/corda-interop-app/interop-workflows/src/main/kotlin/com/weaver/corda/app/interop/flows/WriteExternalStateFlows.kt

```diff
@@ -128,13 +128,12 @@ class GetExternalStateByLinearId(
                 State.Meta.Protocol.CORDA -> {
                     val cordaViewData = ViewDataOuterClass.ViewData.parseFrom(viewDataByteArray)
                     println("cordaViewData: $cordaViewData")
-                    val interopPayload = InteropPayloadOuterClass.InteropPayload.parseFrom(cordaViewData.payload)
+                    val interopPayload = InteropPayloadOuterClass.InteropPayload.parseFrom(cordaViewData.notarizedPayloadsList[0].payload)
```

Comment:
Same comment as earlier.

Reply:
Again this is just a query function to get the external state from vault, which was written to vault only after all the steps of proof verification.